### DISCUSSION TITLE
Can you reload the plotman config?

### DISCUSSION THREAD

**Discussion by Keppto** [ericaltendorf/plotman#492] [Q&A] [answered]
Hello everyone,
I changed the config for plotman but I've already started running plotman plot in the background with the screen command. Is there any way to reload the config so that it would use the new config rather than the old one?
Thanks.

**Comment by cyperbg** [comment, marked as answer]
Press q in Plotman Interactive to quit, then run it again.

**Reply by Keppto** [reply]
Thanks!

**Comment by dragon0087** [comment]
what about running with CLI? Is there anyway to reload the config without killing the current job?

**Reply by Purple-Tentacle** [reply]
As I understand it, Plotman reads the config everytime it is run.
So a `plotman status` should re-read the config.
This will not change the running plot job, of course.

**Reply by mruckli** [reply]
If you’re running `plotman plot` you can safely quit this process without the plotting actually stopping.
So to reload the config just cancel the current process and start again. Once that is done all new plot processes will consider the new config.
